SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/AlwaysInterestedTest/AlwaysInterestedTest.cpp

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "AlwaysInterestedTest.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/AlwaysInterestedTestActors.h"
+
+#include "LoadBalancing/LayeredLBStrategy.h"
+
+#include "Net/UnrealNetwork.h"
+
+/**
+ * This test tests that AlwaysInterested UProperties are replicated to clients and servers correctly.
+ *
+ * The test should include two servers and two clients
+ * The flow is as follows:
+ *  - Setup:
+ *    - Server 1 spawns an actor with AlwaysInterested properties, and two test actors, one of which is set to be AlwaysInterested
+ *  - Test:
+ *    - Move the test actors to server 2 authority area, to ensure interest on server 1 is lost via authority.
+ *    - Validate actors are correctly in or out of view for auth and non-auth servers, and owning and non-owning clients.
+ *    - Move the test actors to server 1 authority area, to ensure interest on server 2 is lost via authority.
+ *    - Validate actors are correctly in the view of auth and non-auth servers.
+ *  - Cleanup:
+ *    - Destroy the actors
+ */
+
+const static float StepTimeLimit = 5.0f;
```

Comment:
this may conflict if we ever switch unity builds on, no? Maybe just move it into PrepareTest preemptively? Performance shouldn't really matter for the prepare step.